packages/wonder-blocks-dropdown/src/components/__tests__/action-menu.test.js

```diff
@@ -537,15 +512,15 @@ describe("ActionMenu", () => {
             );
 
             // Act
-            const opener = menu.find(DropdownOpener).find("button");
+            const opener = screen.getByLabelText("Custom opener");
```

Comment:
We may want to use `getByRole("menu", {name: "Custom opener"}` or whatever the correct role is.  I think that might make the test more robust to possible internal changes to the component.